app/components/ui/molecules/PeoplePicker.js

```diff
@@ -145,19 +145,24 @@ class PeoplePicker extends React.Component {
     this.setState({ searchValue })
   }
 
-  filterPeople() {
+  filterPeople(people) {
     const searchValue = this.state.searchValue.toLowerCase()
-    return this.props.people.filter(person =>
+    return people.filter(person =>
       person.name.toLowerCase().includes(searchValue),
     )
   }
 
   render() {
     const { people, ...otherProps } = this.props
-    const searchOptions = people.map(person => ({ value: person.name }))
+    const sortedPeople = [...people].sort((a, b) =>
+      a.name.localeCompare(b.name),
```

Comment:
Does this run correctly? That following comma looks incorrect...

Reply:
Yes, this runs correctly. That comma was added by either prettier or eslint on the precommit hook, not sure which.